front/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     "next/core-web-vitals",
     "prettier",
     "eslint:recommended",
+    "plugin:@next/next/recommended",
```

Comment:
adding the recommended eslint plugin for next, so we have the same configuration as before